README.md

```diff
@@ -1,9 +1,6 @@
 cmark-gfm
 =========
 
-[![Build Status]](https://travis-ci.org/github/cmark-gfm)
-[![Windows Build Status]](https://ci.appveyor.com/project/github/cmark)
```

Comment:
Should we replace this with [a status badge for the GitHub Actions build][1]?

[1]: https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge